geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/command/GetAll70.java

```diff
@@ -211,16 +211,7 @@ private void fillAndSendGetAllResponseChunks(Region region, String regionName, O
           }
         }
 
-        try {
-          securityService.authorize(Resource.DATA, Operation.READ, regionName, key);
-        } catch (NotAuthorizedException ex) {
```

Comment:
I'm not sure that this can be removed. Might be an issue for backwards compatibility.